#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-python.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Adding python she-bangs.

@DPATCH@

diff -Naur lush-1.1.orig/packages/python/demos/addthem.py lush-1.1/packages/python/demos/addthem.py
--- lush-1.1.orig/packages/python/demos/addthem.py	2003-02-20 21:25:29.000000000 +0000
+++ lush-1.1/packages/python/demos/addthem.py	2005-12-14 13:24:15.508117704 +0000
@@ -1,2 +1,4 @@
+#!/usr/bin/python
+
 def addthree ( x, y, z):
     return x+y+z
diff -Naur lush-1.1.orig/packages/python/demos/addthem2.py lush-1.1/packages/python/demos/addthem2.py
--- lush-1.1.orig/packages/python/demos/addthem2.py	2003-02-20 21:25:31.000000000 +0000
+++ lush-1.1/packages/python/demos/addthem2.py	2005-12-14 13:24:15.508117704 +0000
@@ -1,3 +1,5 @@
+#!/usr/bin/python
+
 class addthemclass:
     def addthree (self, x, y, z):
         return x+y+z
diff -Naur lush-1.1.orig/packages/python/demos/addthemclass.py lush-1.1/packages/python/demos/addthemclass.py
--- lush-1.1.orig/packages/python/demos/addthemclass.py	2003-02-20 21:25:31.000000000 +0000
+++ lush-1.1/packages/python/demos/addthemclass.py	2005-12-14 13:24:15.508117704 +0000
@@ -1,3 +1,5 @@
+#!/usr/bin/python
+
 class addthemclass:
     def addthree (self, x, y, z):
         return x+y+z
diff -Naur lush-1.1.orig/src/date.c lush-1.1/src/date.c
